recipe/run_conda_forge_build_setup_win

```diff
@@ -6,9 +6,7 @@ conda config --set add_pip_as_python_dependency false
 conda update -n root --yes --quiet conda conda-build
 conda install -n root --yes --quiet jinja2 anaconda-client
 
-# Workaround for Python 3.4 and x64 bug in latest conda-build.
-# FIXME: Remove once there is a release that fixes the upstream issue
-# ( https://github.com/conda/conda-build/issues/895 ).
-if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install -n root --yes --quiet conda-build=1.20.0
+# see https://github.com/conda-forge/staged-recipes/pull/750#issuecomment-225165530
+conda install -n root --yes --quiet conda-build=1.20.0
```

Comment:
So, I don't think we can do this downgrade across the board. Otherwise, we are going to need to add back VS2008 64-bit workaround ( https://github.com/conda-forge/staged-recipes/pull/251 ). In the case of `conda-smithy`, we will need to add this for the first time. Could we maybe use `conda-build` 1.20.1 or higher in the other cases at least?